src/components/Toolbar/Toolbar.tsx

```diff
@@ -70,6 +70,11 @@ const Toolbar: FC<ToolbarProps> = ({ onButtonClick }) => {
           text={t("common.join")}
           onClick={() => onToolbarButtonClick("join")}
         />
+        <ToolbarButton
```

Comment:
Use react-router for internal links in the app. So modify ToolbarButton so it accepts link as property and use react-router accordingly.

Reply:
No need for toolbar button anymore. So this is resolved.